v2/concordium/types.proto

```diff
@@ -2363,7 +2405,7 @@ message PendingUpdate {
     // Updates to the transaction fee distribution.
     TransactionFeeDistribution transaction_fee_distribution = 13;
     // Updates to the GAS rewards.
-    GasRewards gas_rewards = 14;
+    GasRewardsCpv0 gas_rewards_cpv_0 = 14;
```

Comment:
I am not a fan of this renaming. You are just causing needless work to everybody using the generated bindings.

Reply:
It's the same above when renaming `GasRewards` to `GasRewardsCpv0`.

Reply:
Changed it back.

Reply:
Are you sure? GasRewardsCpv0 is still there.

Reply:
Fixed.